base/test.jl

```diff
@@ -168,4 +168,20 @@ macro inferred(ex)
     end
 end
 
+#Test equivalence of eigenvectors/singular vectors taking into account possible
```

Comment:
Ah, since you said you had seen it in other packages, I assumed it would be exported. `@test_approx_eq` is documented at least.